src/Components/Login/Login.jsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable import/no-named-as-default, no-shadow, consistent-return */
+import React, { Component } from 'react';
+import { Grid, Card, Button } from 'semantic-ui-react';
+// import { connect } from 'react-redux';
+// import PropTypes from 'prop-types';
+import './Login.css';
+
+// NOTE all commented out code will be used in future mapping of state, dispatching of actions etc
+// import { renderSheets } from '../../actions/renderActions';
+// import injectWidgetId from '../../utils/utils';
```

Comment:
@Gwillison415 I think it's great that you left mention of why this comment is in here, and even did some of the future work preemptively, but I'm not sure if it's a good idea to comment out imports for directories that don't even exist yet. It's easy for me to imagine future developers making different choices about their file structure, based on considerations that are not yet know to us, and therefore I invoke the principle of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it), and ask you to remove the not-yet-needed comments.

As an act of kindness though you might cut an additional branch off of this `stephen` branch before making an additional commit, and notifying the rest of the team they could find some potentially helpful starter code should they begin work on the feature in the future.

Reply:
YAGNI !! ok thanks for the ideas and next steps. will cut after I fix other issues